src/features/rewards/tab/style.ts

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import { Props } from './index'
+
+interface StyleProps {
+  left?: boolean
+  selected?: boolean
+}
+
+export const RewardsTabWrapper = styled<{}, 'div'>('div')`
+  display: flex;
+  font-family: Poppins,sans-serif;
+`
+
+export const StyledSwitch = styled<{}, 'div'>('div')`
+  position: relative;
+  display: block;
+  width: 284px;
```

Comment:
Looks like we're using fixed widths for:
- The component overall
- Each item

This seems problematic for items that have a longer string. Whilst we may only have one usage of this and it only supports 2 Tabs so we can ignore a swathe of detail considerations, we of course support a high number of languages. Can we predict that the word for 'Backup' and 'Restore' will fit in to 135px at 16px Poppins? The first language I looked up (Finnish) had the word for 'Backup' as 'Varmuuskopioida':
![image](https://user-images.githubusercontent.com/741836/47519071-87f11e00-d841-11e8-85fb-c889695c6da6.png)

@jenn-rhim what is the intention here:
- each item takes up the space it needs, and the 'pill' (called 'bullet' here) grows and shrinks as it moves between selected items?
- each item is the same width, taking the largest item's required width
- each item is a fixed size we define keeping 🤞 that we cater to all language string lengths

This is a happy reminder that thoughts and detail like this is why it's important to put effort in to creating a shared spec for a complex custom new thing like this and want to maintain quality @karenkliu @bradleyrichter @jenn-rhim 😃

Reply:
One option for this is to:
- Set a max-width on the overall component (or let it be 100% width of its parent)
- Allow each item to take up the width it needs, but break the word on to the next line if it's too long, even if it's 1-word. So we will have varying width items.
- All items take the same height, so if one item takes up two lines, all the other items take up 2 lines too.

There are many options, but that is 1, and maybe it is not the easiest to implement.

Reply:
@pete thanks for checking a translated string. Until we run into a situation that it becomes a problem, each tab should be given the same width, considering the longest string item, which will provide the visual balance. I don’t expect the word to break into 2 lines if it’s a single word, which in this case it is. Also don’t expect to use multi-word categories for tab headings - tab control probably is not the right choice of the design for that case. There are also other ways to shrink the text. But let’s not worry about it for now. 

Consider using the font size 14 instead of 16. I’d actually prefer the style that leaves more white space within the tab element than filling it.

Reply:
1. The tab component now takes up 100% of the parent width, and divides tab space equally.
2. Text should now truncate at a reasonable limit based on the tab size
3. Font reduced to 14px

Here is the element at a few widths:

500px
<img width="377" alt="screen shot 2018-10-29 at 6 12 45 pm" src="https://user-images.githubusercontent.com/8732757/47689383-f2d68800-dba6-11e8-9076-dc51ab54829e.png">

350px
<img width="526" alt="screen shot 2018-10-29 at 6 12 55 pm" src="https://user-images.githubusercontent.com/8732757/47689391-f702a580-dba6-11e8-9050-a0e6d3677c39.png">

200px
<img width="238" alt="screen shot 2018-10-29 at 6 13 08 pm" src="https://user-images.githubusercontent.com/8732757/47689402-01bd3a80-dba7-11e8-9d59-8321287aaf6a.png">

cc: @petemill @NejcZdovc @jenn-rhim